src/pages/politis/get-started/quick_start.bg.md

```diff
@@ -11,24 +11,51 @@ priority: 2
 
 ## Продукти
 
-Очаквайте скоро.
+Всеки продукт в Политис съдържа информация за име, тип, мерна единица, доставчик. Към продуктите могат да се създават варианти, а самите варианти да се организират в опаковки. Всеки продукт може да бъде зачислен към определена дивизия за по-лесна обработка  и ценообразуване.
+
+<a href="https://docs.clouway.com/bg/politis/guide/product_management/register-product/">Управление на продукти</a>
+<br></br>
 
 ## Складове
 
-Очаквайте скоро.
+Складовете в Политис дава възможност за организиране на местата, от които компанията извършва следните дейности: фактуриране, доставки, складови наличности. 
+
+<a href="https://docs.clouway.com/bg/politis/guide/settings/company-details/">Работни обекти</a>
+<br></br>
 
 ## Документи
 
-Очаквайте скоро.
+В Политис се създават следните и отпечатат документи: 
+
+- *Фактура* 
+- *Кредитно известие*
+- *Дебитно известие*
+- *Стокова разписка*
+- *Списък за опаковане*
+- *Заявка за покупка*
+- *Заявка за трансфер*
+- *Протокол за липси и излишъци*
+
+## Покупка
+
+Политис предлага възможности за стоков контрол при покупка на стоки. Създаване на заявки за покупка, приемането им, както и директно заприхождаване на стока към даден склад спомага за бързо организиране на складови наличности.
+
+<a href="https://docs.clouway.com/bg/politis/guide/stock-control/purchase/purchase-orders/purchase-orders/">Покупка със заявка</a>
```

Comment:
I suppose the gatsby link could be used instead:
https://www.gatsbyjs.org/docs/gatsby-link/

Maybe the i18n support is breaking the navigation and we could wrap link. You may check how the i18n work to be able to see how to use this component for the navigation.